devices/CharacterLcd/LcdValueUnitDisplay.cs

```diff
@@ -0,0 +1,489 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+//using System;
+////using System.Collections.Generic;
+//using System.Globalization;
+//using System.IO;
+////using System.Linq;
+//using System.Reflection;
+//using System.Text;
+//using System.Threading;
+////using System.Threading.Tasks;
+////using Iot.Device.Graphics;
+
+//namespace Iot.Device.CharacterLcd
```

Comment:
I should have a look for that one tomorrow. I have done some work migrating it. I need to check up to which point.

Reply:
@networkfusion here is the pre existing work on CharacterLcd: https://github.com/nanoframework/nanoFramework.IoT.Device/tree/laurent/CharacterLcd

Done quite some time ago. So I guess only code from the font is interesting to look at.

Reply:
Let me take my work into this draft PR